src/utils/hoc/index.ts

```diff
@@ -6,10 +6,10 @@ export const withHooks =
   <H, P, R>(hookFn: HookFn<H, P, R>) =>
   (Component: ComponentType<P>) =>
   (hookParams: H) =>
-  (props: P) => {
-    const results = hookFn(hookParams, props);
-    return createElement(Component, { ...props, ...results });
-  };
+    function (props: P) {
```

Comment:
why did you switch from the initial arrow funciton?

Reply:
I did not. It's some eslint/prettier bullshit again. It happened also on some other component that Jakub noticed.

I actually have no idea which rule makes this change so I can stop it.

I'll revert this in the mean time